test/test_ops.py

```diff
@@ -549,6 +563,14 @@ def test_python_ref_torch_fallback(self, device, dtype, op):
         ],
     )
     @skipIfTorchInductor("Takes too long for inductor")
+    @skipOps(
+        "TestCommon", "test_python_ref_executor", \
+            (('_refs.mul', '', 'xpu', (torch.complex32,), False),), all_opinfos=python_ref_db
+    )
+    @skipOps(
+        "TestCommon", "test_python_ref_executor", \
+            (('_refs.pow', '', 'xpu', (torch.complex32,), False),), all_opinfos=python_ref_db
+    )
```

Comment:
Why do we need to add the two skips here?

Reply:
The python_ref_db defines a Decorator of expected failure for the two ops, but we got passed. So it reports 'Unexpected success'.